### PR TITLE
Use tailwind

### PR DESCRIPTION
**Pull request by quick007**
I developed this with out seeing the live preview but it probably is fine

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 module.exports = {
   content: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      maxWidth: {
+        "fold-lines": "calc(80rem+200px)",
```
Comment (mikefortney) [inline tailwind.config.js:6]: this doesn't seem to be working, as the width of the div is screen width, not 80rem + 200px

**Review by mikefortney — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (quick007) [reply tailwind.config.js:6]: I was just converting the original CSS, not fixing it (mainly cause I couldn't get live preview working)
image

Comment (mikefortney) [reply tailwind.config.js:6]: I see. Well, the width was working before. So please either get the width working or return the width to inline style. Thanks!

**Comment by kurdybacha** [comment]
@quick007 I will close this PR for now. Feel free to reopen when you have time to address the comments.

**Pull request closed without merging**